docs/requirements.txt

```diff
@@ -0,0 +1,8 @@
+-f http://simple.crate.io/
```

Comment:
seems that this was an alternative package index. at least in 2013... removing this